build/tasks/spec-task.coffee

```diff
@@ -104,6 +104,30 @@ module.exports = (grunt) ->
 
       callback(null, error)
 
+  runMainProcessSpecs = (callback) ->
+    appPath = getAppPath()
+    resourcePath = process.cwd()
+    mainProcessSpecsPath = path.resolve('spec/browser')
```

Comment:
The main reason was that we store main process code under `src/browser`, so I thought I'd name the spec folder accordingly. Also `main` sounds a bit generic, so I guess we could rename the `browser` folders to `main-process` and have everything named consistently.

Thanks, @kuychaco!